### ISSUE TITLE
Question: Extracting the transcribed output

### ISSUE THREAD

**Issue by prashantserai** [speechbrain/speechbrain#334]
Hi!
I'm using a model trained on Librispeech with the ASR seq2seq recipe to transcribe a certain set of wav files. I want to dump the ASR decoded results to a csv file. 

**Questions:**
1. I added a line ```stats["word_seq"] = word_seq``` at the end of the compute_objective function, is this the final output? The WER on my out of domain test-set seems a bit too bad, hence I ask.
2.  in the prepared csv file for the test set, the transcripts provided (in the "wrd" column) should only matter for reporting WER and have no effect on the transcribed ASR output, correct? (I'm using dummy transcripts at the moment).

PS: My audio files have a sample rate of 22050 Hz and I'm working on resampling them to 16khz, for better consistency with the training set of Librispeech.
